engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -332,7 +332,7 @@ <h4 translate> Published Repository Information </h4>
       <div class="checkbox" ng-hide="repository.content_type === 'docker' || repository.content_type === 'ansible_collection'">
         <label>
           <input id="unprotected" name="unprotected" ng-model="repository.unprotected" type="checkbox"/>
-          <span translate>Publish via HTTP</span>
+          <span translate>Uprotected</span>
```

Comment:
It might be good to add a description to this, something like  'Do not require a subscription entitlement certificate for accessing this repository'